src/main/scala/org/apache/spark/sql/kinesis/ShardSyncer.scala

```diff
@@ -17,11 +17,15 @@
 
 package org.apache.spark.sql.kinesis
 
+import scala.util.parsing.json._
```

Comment:
This will give a styling error. Can we include this along with other scala imports?